code/modules/projectiles/ammunition/ballistic/pistol.dm

```diff
@@ -48,3 +48,10 @@
 	caliber = ".50"
 	projectile_type = /obj/item/projectile/bullet/a50AE
 
+/obj/item/ammo_casing/caseless/lathe
+	name = "glass metal alloy round"
```

Comment:
Consider something like "glass-metal" or "glass and metal" or just "alloy round", since this reads as a glass version of a metal alloy round currently.